raiden/api/v1/encoding.py

```diff
@@ -338,6 +338,7 @@ class PaymentSchema(BaseSchema):
     identifier = fields.Integer(missing=None)
     secret = SecretField(missing=None)
     secret_hash = SecretHashField(missing=None)
+    lock_timeout = fields.Integer(missing=None)
```

Comment:
Please add a documentation for this in the [`Payment` section](https://github.com/raiden-network/raiden/blob/develop/docs/rest_api.rst#payments) of the docs: